Makefile

```diff
@@ -386,7 +390,37 @@ helm-chart: manifests kustomize helmify
 unit-test: ## Run unit tests for the helm chart
 	PATH="$(CURDIR)/scripts:$(PATH)" bats test/unit/
 
+##@ GKE
+
+.PHONY: create-gke
+create-gke: gcloud ## Create a new GKE cluster
+	$(TERRAFORM) -chdir=$(TF_GKE_DIR) init -upgrade
+	$(TERRAFORM) -chdir=$(TF_GKE_DIR) apply -auto-approve
+	$(GCLOUD) container clusters get-credentials $$($(TERRAFORM) -chdir=$(TF_GKE_DIR) output -raw kubernetes_cluster_name) \
+	--region $$($(TERRAFORM) -chdir=$(TF_GKE_DIR) output -raw region)
+
+include $(TF_GKE_DIR)/output.env
```

Comment:
@benashz - I'm loading the terraform outputs this way, I've also committed an empty env file so make doesn't throw an error.